.gitignore

```diff
@@ -3,6 +3,9 @@ node_modules
 /build
 /coverage
 packages/*/dist
+packages/**/*.js
+packages/**/*.d.ts
+packages/**/*.js.map
```

Comment:
Can we ignore _all_ `.map` files? E.g.: `packages/**/*.map`

Presumably we'll need to negate this rule in `.npmignore` so we still publish `.map` files to npm.

Reply:
We actually only `.npmignore` `.ts` files, so this won't be an issue for `npm`.

And yes I will change to just ignore all `.map` files

Reply:
npmignore _overrides_ gitignore. You shouldn't need to go out of your way to override gitignore in npmignore - npm only checks one or the other.

On the other hand, if there's anything that we _should_ be ignoring that's in our gitignore and not our npmignores, we should fix that...